oak_utils/src/io.rs

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use anyhow::Context;
+use std::{
+    fs::File,
+    io::{Read, Write},
+};
+
+/// Read the contents of a file.
+pub fn read_file(filename: &str) -> anyhow::Result<Vec<u8>> {
+    let mut file =
+        File::open(filename).with_context(|| format!("Failed to open file <{}>", filename))?;
+    let mut buffer = Vec::new();
+    file.read_to_end(&mut buffer)
+        .with_context(|| format!("Failed to read file <{}>", filename))?;
+    Ok(buffer)
```

Comment:
I think https://doc.rust-lang.org/std/fs/fn.read.html does this already

Reply:
Done